src/main/resources/application.properties

```diff
@@ -8,4 +8,4 @@ spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.H2Dialect
 spring.jpa.hibernate.ddl-auto=update
 spring.h2.console.enabled=true
-spring.h2.console.path=/h2-console
+spring.h2.console.path=/h2-console
```

Comment:
the same line is committed, if there was no change, should be reverted before committing.